### PR TITLE
fix: styles should now be scoped and only affect installer

### PR DESCRIPTION
**Pull request by vojtechsimetka**

### CODE REVIEW

**Review by Cafe137 — APPROVED** [review]

File: ui/src/react-app-env.d.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```
Comment (AuHau) [inline ui/src/react-app-env.d.ts:1]: Why is this here? 😳

File: ui/src/installer/index.css [hunk 2]
```diff
@@ -7,20 +7,20 @@ body {
     margin: 0;
 }
 
-body,
-button {
+.installer body,
```
Comment (AuHau) [inline ui/src/installer/index.css:10]: Well, at least this one does not make sense, as you would have to put the `installer` class on `html` tag in order for this to work... I would propose to have it as part of the `installer` class then.
```suggestion
.installer,
```

**Review by AuHau — CHANGES_REQUESTED** [review]

File: ui/src/react-app-env.d.ts (hunk 1 above)
Comment (vojtechsimetka) [reply ui/src/react-app-env.d.ts:1]: This needs to be there to support importing SVGs. It brings in the SVG types etc.

Comment (AuHau) [reply ui/src/react-app-env.d.ts:1]: Ok, I will take your word for it :-)

**Review by AuHau — APPROVED** [review]

File: ui/src/react-app-env.d.ts (hunk 1 above)
Comment (vojtechsimetka) [reply ui/src/react-app-env.d.ts:1]: Well you can try erasing it :)

Comment (vojtechsimetka) [reply ui/src/react-app-env.d.ts:1]: Actually here https://stackoverflow.com/a/71153426
